source/release-notes/2.9.1.rst

```diff
@@ -0,0 +1,46 @@
+.. _PMM-2.9.1:
+
+================================================================================
+*Percona Monitoring and Management* 2.9.1
+================================================================================
+
+:Date: August 3, 2020
+:Installation: `Installing Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html>`_
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_)
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+Improvements
+================================================================================
+
+* :jirabug:`PMM-6230`: Home Dashboard: Customizations retained across updates
+* :jirabug:`PMM-6300`: Query Analytics Dashboard: Column sorting arrows made easier to use (Thanks to user debug for reporting this issue)
+* :jirabug:`PMM-6208`: Security Threat Tool: Silence viewed but unactioned alerts
```

Comment:
Alert will appear in 24h is the source of the problem will not be fixed. so this is temporary  hide the alert from the panel